OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -1109,5 +1120,350 @@ public void TestGetVariationBucketingIdAttribute()
         }
 
         #endregion
+
+        #region Test GetFeatureVariable<Type> Typecasting
+
+        [Test]
+        public void TestGetFeatureVariableBooleanReturnTypecastedValue()
+        {
+            var featureKey = "featureKey";
+            var variableKeyTrue = "varTrue";
+            var variableKeyFalse = "varFalse";
+            var variableKeyNonBoolean = "varNonBoolean";
+            var variableKeyNull = "varNull";
+            var featureVariableType = FeatureVariable.VariableType.BOOLEAN;
+
+            OptimizelyMock.Setup(om => om.GetFeatureVariableValueForType(It.IsAny<string>(), variableKeyTrue, It.IsAny<string>(),
+                It.IsAny<UserAttributes>(), featureVariableType)).Returns("True");
```

Comment:
Please add a test case for "true" as we have all lowercase letter true in data file